bigtable/client/internal/table_admin.h

```diff
@@ -30,6 +30,17 @@ namespace noex {
  */
 class TableAdmin {
  public:
+  template <typename T, typename Parameter>
```

Comment:
Thanks for creating this class!  Do you mind moving this to its own file?  And it needs a comment describing what it does.